### DISCUSSION TITLE
How to add a custom field to directus_users that also shows in graphql

### DISCUSSION THREAD

**Discussion by mattiaz9** [directus/directus#13985] [Archive]
I added a custom field `username` to the table `directus_users` but it doesn't show up in graphql.

I need the username for the author page of my blog because the email doesn't make sense for the url, so I need the username in graphql for filtering.

Any help on how can I achieve this?

**Comment by azrikahar** [comment]
It seems to work on my end:

https://user-images.githubusercontent.com/42867097/174535748-c07d7e6e-53d3-4481-80f1-f33a82d6e22d.mp4

(also tested in REST which also works, presumably your "doesn't show up in graphql" implies it works on REST for you)

Perhaps it's caching related and somehow your schema is not up to date? Can you try using the [clear cache util endpoint](https://docs.directus.io/reference/system/utilities/#clear-the-internal-cache) to rule this out?

**Comment by mattiaz9** [comment]
My bad! I had some custom restrictions I forgot about. Thank you anyway :)
